ingest/sqs/domain/telemetry.go

```diff
@@ -27,4 +27,21 @@ var (
 	// * msg - the error returned
 	// * height - the height of the block being processed
 	SQSProcessBlockPanicMetricName = "sqs_process_block_panic"
+
+	// sqs_process_block_duration
+	//
+	// histogram that measures the duration of processing a block
+	//
+	// Has the following labels:
+	// * height - the height of the block being processed
+	SQSProcessBlockDurationMetricName = "sqs_process_block_duration"
+
+	// sqs_grpc_connection_error
+	//
+	// counter that is increased if grpc connection fails
+	//
+	// Has the following labels:
+	// * err - the error returned
+	// * height - the height of the block being processed
+	SQSGRPCConnectionErrorMetricName = "sqs_grpc_connection_error"
```

Comment:
Note: commenting to link new alerts in this Linear task:
https://linear.app/osmosis/issue/STABI-70/instrument-new-metrics-for-stage-2-sqs-grpc-refactor